lci_strategic_plugin/src/lci_strategic_plugin.cpp

```diff
@@ -705,7 +704,7 @@ void LCIStrategicPlugin::planWhenAPPROACHING(const cav_srvs::PlanManeuversReques
 
       if (early_arrival_time_green_et.toSec() - nearest_green_signal_start_time.toSec() < config_.green_light_time_buffer)
       {
-        nearest_green_entry_time_cached_ = nearest_green_signal_start_time + ros::Duration(config_.green_light_time_buffer);
+        nearest_green_entry_time_cached_ = nearest_green_signal_start_time + ros::Duration(config_.green_light_time_buffer + 0.01); 
```

Comment:
If there is already a buffer why do we need to add an extra 0.01 to it? Why not just increase the buffer parameter?

Reply:
Actually including in the buffer doesn't work because it uses that same buffer to check early and late. If buffer is 2s and green starts at 10s, it will check +/-2s from 12s. If the buffer was 2.01s and green starts at 10s again, it checks +/-2.01 from 12.01, so both checks 10s. Which is why 0.01 is hardcoded

Reply:
Ok @aswath1 let's add that as an epsilon constant in the file with a comment describing its use.